src/js/view/controls/controlbar.js

```diff
@@ -275,7 +275,10 @@ export default class Controlbar {
         _model.change('fullscreen', this.onFullscreen, this);
         _model.change('streamType', this.onStreamTypeChange, this);
         _model.change('dvrLive', (model, dvrLive) => {
-            toggleClass(this.elements.live.element(), 'jw-dvr-live', dvrLive === false);
+            const liveElement = this.elements.live.element();
+
+            toggleClass(liveElement, 'jw-dvr-live', !dvrLive);
+            setAttribute(liveElement, 'aria-label', !dvrLive ? localization.notLive : localization.liveBroadcast);
```

Comment:
```
const { liveBroadcast, notLive } = localization;
dvrLive ? liveBroadcast : notLive;
```
notice the change in order; positive conditions are easier to read ( `dvrLive` vs `!dvrLive`)